packages/@aws-cdk/aws-chatbot/lib/slack-channel-configuration.ts

```diff
@@ -303,5 +312,17 @@ export class SlackChannelConfiguration extends SlackChannelConfigurationBase {
     this.slackChannelConfigurationArn = configuration.ref;
     this.slackChannelConfigurationName = props.slackChannelConfigurationName;
   }
+
+  /**
+   * Adds a SNS topic that deliver notifications to AWS Chatbot.
+   * @param notificationTopic
+   */
+  public addNotificationTopic(notificationTopic: sns.ITopic): void {
+    this.notificationTopics.push(notificationTopic);
+  }
+
+  private renderTopics(): string[] {
+    return this.notificationTopics?.map(topic => topic.topicArn) || [];
```

Comment:
Hmm, why the `.?`? `this. notificationTopics` has type `sns.ITopic[]`, so it can never be `undefined`.

It will also allow you to get rid of the `|| []` part.

Reply:
My bad. I started creating this array as optional and I forgot to change it. Done